src/Containers/VendorForm/VendorForm.scss

```diff
@@ -79,23 +81,42 @@
 .vendor-name-input {
   width: 75%;
   margin-top: 10px;
-  border-radius: 20px;
-  border: 3px solid $dark_red_border_color;
-  background-color: $light_red_fill_color;
+  // border-radius: 20px;
```

Comment:
We shouldn't get rid of the commented out code. I think the rehaul of this form is a bit much - I didn't realize the extent to which you were planning on revamping the whole thing. I don't think the opaqueness works well with the form, especially on the right side - it is difficult to tell that those are inputs and there is just too much going on with that background image coming through so much. The back icon is also very hard to see because of the background. The create vendor button looks pretty out of place now too